test/mciproject.yml

```diff
@@ -8,6 +8,18 @@ functions:
     command: git.get_project
     params:
       directory: wiredtiger
+  "fetch artifacts" : &fetch_artifacts
+    - command: s3.get
+      params:
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        remote_file: wiredtiger/${build_variant}/${revision}/artifacts/${build_id}.tgz
+        local_file: wiredtiger.tgz
+        bucket: build_external
+    - command: shell.exec
+      params:
+        script: |
+          tar -xvf wiredtiger.tgz
```

Comment:
Tar wierdness, the {{z}} is implicit. However, you're right and I should be explicit.